src/HealthySqlDump.php

```diff
@@ -141,6 +145,18 @@ protected function failsOnNoSql($backup)
         }
     }
 
+    /**
+     * @param Backup $backup
+     * @param int $timesBackupSize
+     * @throws DatabaseImportFailed
+     */
+    public function failsOnLackOfDiskSpace(Backup $backup, $timesBackupSize = 4)
```

Comment:
Not sure if `failsOnNotEnoughDiskSpace` would been a better name for this method?

`$timesBackupSize` is set as a variable mostly because it helps making the test easier, but more than happy to see how if this can be done in other ways. Could also be an configuration option for the whole health check, like we have with `timeout`

Reply:
Check naming suggestion in previous comment.

Check implementation in later comment. This should allow it to be protected as well.

As for the `$timesBackupSize` I believe this should also be unnecessary as an argument now.

Reply:
```php
$estimatedSqlSize = $backup->size() * 11;
$requiredSpace = $estimatedSqlSize * 2;
```